testData/solutions/win32.json5

```diff
@@ -1,4 +1,6243 @@
+                                "Red",
+                                "Yellow",
+                                "Green",
+                                "Cyan",
+                                "Blue",
+                                "Magenta",
+                                "White"
+                            ],
+                            "default": 255
+                        }
+                    },
+                    "FocusScheme.FocusENHThickness": {
+                        "schema": {
+                            "title": "Focus locator thickness",
+                            "description": "Specifies the thickness of the focus locator.",
+                            "enum": [
+                                1,
+                                5,
+                                9
+                            ],
+                            "enumLabels": [
+                                "Thin",
+                                "Medium",
+                                "Thick"
+                            ],
+                            "default": 5
+                        }
+                    },
+                    "FocusScheme.FocusENHTransparency": {
+                        "schema": {
+                            "title": "Focus locator transparency",
+                            "description": "Specifies the transparency of the focus locator. Range goes from 'Clear' to 'Solid'.",
+                            "type": "integer",
+                            "minimum": 31,
+                            "maximum": 255,
+                            "default": 143
+                        }
+                    },
+                    "FocusScheme.FocusENHPadding": {
+                        "schema": {
+                            "title": "Focus locator margin",
+                            "description": "Specifies the margin for the focus locator. Range goes from 'Narrow' to 'Wide'.",
+                            "type": "integer",
+                            "minimum": 1,
+                            "maximum": 9,
+                            "default": 1
+                        }
+                    },
+                    "FocusScheme.FocusENHMode": {
+                        "schema": {
+                            "title": "Focus locator display mode",
+                            "description": "Specifies how and when the focus locator is displayed.",
+                            "enum": [
+                                2,
+                                1,
+                                3
+                            ],
+                            "enumLabels": [
+                                "Continuosly",
+                                "Briefly",
+                                "When the modifier key is pressed"
+                            ],
+                            "default": 2
+                        }
+                    },
+                    // Bitmask setting
```

Comment:
I'd suggest searching for the label and/or comment and just ditching all of these bitmask fields.

Reply:
These have all been removed as part of my cleanup based on your improved WARNING's from the SR, expect for one that had a legit default, minimum, and maximum value, so I guess I'll leave it for now?

Reply:
Yes, I have been discussing leaving whatever we can that's not obviously dangerous.  I'm fine with seldom-used settings as long as they aren't foot cannons like the unbounded/defaultless ones.